setup.py

```diff
@@ -79,7 +79,13 @@ def str_from_file(name):
     # License: Apache 2.0
     "yappi==1.2.3",
     # License: BSD
-    "ijson==2.6.1"
+    "ijson==2.6.1",
+    # License: Apache 2.0
+    # transitive dependencies:
+    #   google-crc32c: Apache 2.0
+    "google-resumable-media==1.1.0",
```

Comment:
You also need to update the script `create-notice.sh` accordingly. I also wonder whether we should investigate to make S3 and GCS optional dependencies and avoid pulling them in core Rally. I'd be ok if we tackle this separately though.

Reply:
Great catch. Done in 27419b61228b130511f2ced2e08f6bc32ac37a42 (where I also fixed the URL for `yarl` which was actually downloading html).
As discussed offline, will create an issue for making the S3 dependencies optional.